community-operators/opsmx-spinnaker-operator/opsmx-spinnaker-operator.v1.13.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,160 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
```

Comment:
Please add the `metadata.annotations.repository` field to the CSV. This should be a URL to the operator's source code repository.